lib/reek/core/ast_node.rb

```diff
@@ -14,6 +14,10 @@ def comments
         @comments.map(&:text).join("\n")
       end
 
+      def comments_array
```

Comment:
I introduced it because `#comments` was already taken and I didn't want to break everything immediately. I'll refactor.